### PR TITLE
Add support for cookies in PoW

### PR DESCRIPTION
**Pull request by agusaldasoro**
1. Use `utf8` encoding for hashing algorithm.
2. Clear JWT cookie when invalidated.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -184,7 +184,7 @@ services:
 
 
   pow-auth-server:
-    image: decentraland/pow-authorization-server
+    image: decentraland/pow-authorization-server:b95a718abb23bb51287b373f5f1d7ebe8173e637
```
Comment (agusaldasoro) [inline docker-compose.yml:187]: This will change when latest version of pow-server is released

File: local/nginx/lua/clean_jwt_cookie.lua [hunk 2]
```diff
@@ -5,7 +5,9 @@ if not cookie then
     ngx.log(ngx.ERR, err)
 end
 
-local ok, err = cookie:set({key = "JWT", value = ""});
+-- Set empty with an expires in the past
+local ok, err = cookie:set({key = "JWT", value = "; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT"});
```
Comment (agusaldasoro) [inline local/nginx/lua/clean_jwt_cookie.lua:9]: This was necessary for the Browser to clean the JWT cookie which was invalidated.

**Review by menduz — COMMENTED** [review]
Looking good

**Review by jmoguilevsky — APPROVED** [review]

**Comment by jmoguilevsky** [comment]
Just put the latest version of pow server  in the docker compose when the release is ready

**Pull request merged**